scripts/broad_dsde_workflows/GenotypeGVCFsScatterWf_170204.wdl

```diff
@@ -0,0 +1,173 @@
+## Copyright Broad Institute, 2017
+## 
+## This WDL workflow runs GenotypeGVCFs on a set of GVCFs to joint-call multiple 
+## samples, scattered across intervals.
+##
+## Requirements/expectations :
+## - GVCFs produced by HaplotypeCaller in GVCF mode 
+##
+## Outputs :
+## - A (typically but not necessarily multisample) VCF file and its index
+##
+## Cromwell version support 
+## - Successfully tested on v24
+## - Does not work on versions < v23 due to output syntax
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation. 
+## For program versions, see docker containers. 
+##
+## LICENSING : 
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in 
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may 
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker 
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+# TASK DEFINITIONS
+
+# Unzip GVCFs because GenotypeGVCFs is picky
+task UnzipGVCF {
+    File gzipped_gvcf
+    String unzipped_basename
+    Int disk_size
+    String mem_size
+
+    # HACK ALERT! Using .gvcf extension here to force IndexFeatureFile to make the right 
```

Comment:
Mainly because this should be a very temporary situation, so when it's fixed we can just change this bit. In fact I should check whether it's still even the case.